Voycar.Api.Web/Features/Members/Services/EmailService/EmailService.cs

```diff
@@ -51,8 +51,21 @@ public void SendVerificationEmail(Member member)
     }
 
 
+    public void SendPasswordResetEmail(User user)
+    {
+        var email = this.CreatePasswordResetEmail(user, GeneratePasswordResetLink());
+        this.SendEmail(email);
+    }
+
+
+    // todo Link to Frontend must be added + VerificationToken must be attached
```

Comment:
`// todo`?

Reply:
Der Link muss später anders, wenn wir ein Frontend haben. Da wir dies noch nicht haben, habe ich ein Todo hingesetzt für später. Kann die To-dos auch herausnehmen und nur den Kommentar hinschreiben. Bei den anderen Todos ähnlich.

Reply:
Ah okay. Dann sind die `ToDo`s die irgendwie später noch fürs Frontend angepasst werden müssen okay so. Aber dann bitte überall einheitlich `ToDo` schreiben, und nicht manchmal `todo`.